src/Stryker.Core/Stryker.Core/Helpers/RoslynHelper.cs

```diff
@@ -56,6 +49,13 @@ baseMethod switch
         public static bool NeedsReturn(this AccessorDeclarationSyntax baseMethod) =>
             baseMethod.Keyword.Text == "get";
 
+        /// <summary>
+        /// Checks if a member is static
+        /// </summary>
+        /// <param name="node">Member to analyze</param>
+        /// <returns>true if it is a static method, properties, fields...</returns>
+        public static bool IsStatic(this MemberDeclarationSyntax node) => node.Modifiers.Any(x => x.Kind() == SyntaxKind.StaticKeyword);
```

Comment:
This file is part of a difficult merge conflict between master and V1. It is currently being solved by @dupdob and I think also includes this change but to be safe I would prefer if you could revert the changes in this file.

Reply:
I've reverted all changes on this file to avoid merge conflicts.